### PR TITLE
Clear Diagnostics on TextDocument close.

### LINKED ISSUE #153: Clear diagnostics on document close
Currently when a document is closed, the language server doesn't clear diagnostics for the document.

### PR DESCRIPTION
**Pull request by rebornix**
Fix #153.

### CODE REVIEW

File: jedi_language_server/initialization_options.py [hunk 1]
```diff
@@ -38,6 +38,7 @@ class Completion(Model):
 
 class Diagnostics(Model):
     enable: bool = True
+    did_close: bool = True
```
Comment (pappasam) [inline jedi_language_server/initialization_options.py:41]: Per your recommendation [here](https://github.com/pappasam/jedi-language-server/issues/153#issuecomment-875844190), we will want to remove the `did_close` option from `initialization_options.py` and instead rely on the already-existing `enable` option. Eg, we'll always clear diagnostics as long as diagnostics are enabled by the user. In short, I recommend that we remove this option.

**Review by pappasam — CHANGES_REQUESTED** [review]
One small change!

File: jedi_language_server/initialization_options.py (hunk 1 above)
Comment (rebornix) [reply jedi_language_server/initialization_options.py]: thanks, I removed the option.

**Comment by rebornix** [comment]
@pappasam updated per review.

**Review by pappasam — APPROVED** [review]
lgtm!

**Comment by rebornix** [comment]
@pappasam thanks for the quick review!

**Pull request merged**